Terminal.Gui/Core/Toplevel.cs

```diff
@@ -171,7 +177,35 @@ public override bool ProcessKey (KeyEvent keyEvent)
 			return false;
 		}
 
-		///<inheritdoc/>
+		IEnumerable<View> GetToplevelSubviews (bool isForward)
```

Comment:
This crashes if the Toplevel is a dialog and the user presses Tab.

Reply:
Oh sorry. I already fixed.